### ISSUE TITLE
`@psalm-assert` error upstream due to `ValidatorChain`

### ISSUE THREAD

**Issue by internalsystemerror** [laminas/laminas-validator#138]
### Bug Report

|    Q        |   A
|------------ | ------
| Version(s)  | >2.18.0

#### Summary

The following line introduces an error upstream:
https://github.com/laminas/laminas-validator/blob/9fa954d60065913c165318b0c1957050794867ec/src/ValidatorChain.php#L83.

#### Current behavior

Running psalm on laminas/laminas-form we get the error:
```
ERROR: InvalidDocblock - src/FormAbstractServiceFactory.php:144:58 - Variable $this is not an argument so cannot be asserted (see https://psalm.dev/008)
        $inputFilterFactory->getDefaultValidatorChain()->setPluginManager($container->get('ValidatorManager'));
```

#### How to reproduce

Run static analysis as stated above.

#### Expected behavior

No error

**Comment by internalsystemerror** [comment]
I think this appears to be more of a bug in psalm than laminas-validator, as:

```
        $validatorChain = $inputFilterFactory->getDefaultValidatorChain();
        $validatorChain->setPluginManager($container->get('ValidatorManager'));
```

does not give the same error
